codes/dart/chapter_array_and_linkedlist/array.dart

```diff
@@ -0,0 +1,110 @@
+/**
+ * File: array.dart
+ * Created Time: 2023-01-20
+ * Author: Jefferson (JeffersonHuang77@gmail.com)
+ */
+
+import 'dart:math';
```

Comment:
Could we create a `utils/include.dart` and import all the common libs in this file?

Reply:
No, this way will throw an error.